linodecli/__init__.py

```diff
@@ -308,7 +308,7 @@ def main():
 
         print()
         print("To reconfigure, call `linode-cli configure`")
-        print("For comprehensive documentation, visit https://developers.linode.com")
+        print("For comprehensive documentation, visit https://www.linode.com/docs/api/openapi.yaml")
```

Comment:
This one was intended to be a link to human-readable documentation.  Is this the right address?

```suggestion
        print("For comprehensive documentation, visit https://www.linode.com/docs/api/")
```

Reply:
Thanks @Dorthu! Updated.